src/client/components/componentWithSliders/allSliders/allBlocks.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import './alllBlocks.css';
+
+import BlockOfSliders from './blockOfSliders';
+import PlayButton from './playButton';
+import StopButton from './stopButton';
+
+import BLOCKS from '../../../helpers/blocks';
+import Pizzicato from "pizzicato";
+
+
+class AllBlocks extends Component {
+
+  constructor(props, context){
+    super(props, context);
+    this.state = {
+      sound: new Pizzicato.Sound({
+        source: 'file',
+        options: { path: require('../../../sounds/sound.mp3') }
+        },
+        () => BLOCKS.forEach(({createEffect}) => {
+          this.state.sound.addEffect(createEffect);
+            })
+      ),
+      playing: false,
+    };
+  }
+
+  play = () => {
+    this.setState({
+      playing: !this.state.playing});
```

Comment:
please, also fix code formatting here, for eg.,
` this.setState({ playing: !this.state.playing });`  or 
```
` this.setState({
      playing: !this.state.playing
});`
```
but for your case, i would suggest first example, because you have only 1 property

Reply:
fixed